etc/docker/test/matrix_policy_package_tests.yml

```diff
@@ -0,0 +1,93 @@
+atlas:
+  installation_cmd: ""
+  config_overrides: 
+    permission: atlas
+    extract_scope:  atlas
+    schema: atlas
+    lfn2pfn_algorithm_default: "hash"
+  tests:
+    allow:
+      - rucio_tests
+    deny:
```

Comment:
Every time a new test file is created, it needs to be added here. This seems to add a constraint nobody will remember in the future. Can we change this to only specify the allowed ones?

Reply:
These test files will be migrated to use the newly created factories.